### PR TITLE
CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RS-labhub**
## Fixes Issue
Closes #2063

## Changes proposed

Fixed grammatical mistakes

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @RS-labhub. Welcome to LinksHub! :) We don't accept pull requests that are unassigned, so I'm putting yours in draft mode for now. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊

**Comment by RS-labhub** [comment]
> Hi @RS-labhub. Welcome to LinksHub! :) We don't accept pull requests that are unassigned, so I'm putting yours in draft mode for now. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊

So documentation is not accepted?

**Comment by CBID2** [comment]
> > Hi @RS-labhub. Welcome to LinksHub! :) We don't accept pull requests that are unassigned, so I'm putting yours in draft mode for now. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊
> 
> So documentation is not accepted?
We do @RS-labhub. You just have not raised an issue for it. Refer to the Contributing Guide like I previously suggested.

**Comment by RS-labhub** [comment]
> > > Hi @RS-labhub. Welcome to LinksHub! :) We don't accept pull requests that are unassigned, so I'm putting yours in draft mode for now. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊
> > 
> > 
> > So documentation is not accepted?
> > We do @RS-labhub. You just have not raised an issue for it. Refer to the Contributing Guide like I previously suggested. That way, the maintainers can review it, assign it to you, and review this pull request.


> > > Hi @RS-labhub. Welcome to LinksHub! :) We don't accept pull requests that are unassigned, so I'm putting yours in draft mode for now. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊
> > 
> > 
> > So documentation is not accepted?
> > We do @RS-labhub. You just have not raised an issue for it. Refer to the Contributing Guide like I previously suggested. That way, the maintainers can review it, assign it to you, and review this pull request.

Oh okay okayy

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,8 +108,8 @@ Follow these steps to setup LinksHub on your local machine
 
 #### 🌟Follow these steps to add your Category/Subcategory to the Hub
 
-- To add a newly created category or subcategory, include them in the [data.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/data.ts) & [index.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/index.ts) files located in the `database` folder.
-- Each category has its own folder with its name
+- To add a newly created category or sub-category, include them in the [data.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/data.ts) & [index.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/index.ts) files located in the `database` folder.
```
Comment (CBID2) [inline CONTRIBUTING.md:111]: [proofreading] "subcategory" is spelled correctly, so no need for changes here 
```suggestion
- To add a newly created category or subcategory, include them in the [data.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/data.ts) & [index.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/index.ts) files located in the `database` folder.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ Follow these steps to setup LinksHub on your local machine
 
       > For example, if you are adding a category called `open source`, add it to the `export type` statement in the appropriate style.
 
-  - If you are adding a subcategory, go to `components/TopBar/CategoryDescriptions.ts` and add the subcategory name used when exporting the name in `database/index.ts`, along with a concise description in the correct format.
+  - If you are adding a subcategory, go to `components/TopBar/CategoryDescriptions.ts` and add the sub-category name used when exporting the name in `database/index.ts`, along with a concise description in the correct format.
```
Comment (CBID2) [inline CONTRIBUTING.md:133]:
```suggestion
  - If you are adding a subcategory, go to `components/TopBar/CategoryDescriptions.ts` and add the subcategory name used when exporting the name in `database/index.ts`, along with a concise description in the correct format.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -145,13 +145,13 @@ Follow these steps to setup LinksHub on your local machine
 
 #### 🌟Follow these steps to add your links to the Hub
 
-- Choose the category and subcategory under which you want to add the resource.
+- Choose the category and sub-category under which you want to add the resource.
```
Comment (CBID2) [inline CONTRIBUTING.md:148]:
```suggestion
- Choose the category and subcategory under which you want to add the resource.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -192,14 +192,14 @@ When adding _YouTube_ channel links, please specify _the language_ of the channe
...
-- Add or update the description in front of the subcategory name.
+- Add or update the description in front of the sub-category name.
 
   > For example, to add the description of the `fonts` subcategory under the `frontend` category, you would do the following:
 
   ```js
   const categoryDescriptions: CategoryDescriptions = {
     //frontend
-    -                      //other subcatogery
+    -                      //other sub-catogery
```
Comment (CBID2) [inline CONTRIBUTING.md:202]:
```suggestion
    -                      //other subcategory
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -145,13 +145,13 @@ Follow these steps to setup LinksHub on your local machine
 
 #### 🌟Follow these steps to add your links to the Hub
 
-- Choose the category and subcategory under which you want to add the resource.
+- Choose the category and sub-category under which you want to add the resource.
 
 - Locate the appropriate structure: `database/category_name/subcategory_name.json`
 
-> For example, to add a resource in the `fonts` subcategory under the `frontend` category, navigate to `database/frontend/fonts.json`.
+> For example, to add a resource in the `fonts` sub-category under the `frontend` category, navigate to `database/frontend/fonts.json`.
```
Comment (CBID2) [inline CONTRIBUTING.md:152]:
```suggestion
> For example, to add a resource in the `fonts` subcategory under the `frontend` category, navigate to `database/frontend/fonts.json`.
```

**Comment by CBID2** [comment]
Hi @RS-labhub. Thanks for contributing! :) Turns out subcategory is one word and the only issue there is a period, so I'm going to close this PR as merging that one error does not change too much. I highly recommend raising a different issue for this project. If you need ideas, check out [this section of the contributing guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#how-to-contribute-).

**Pull request closed without merging**